spec/support/shipment_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class ShipmentDecorator < SimpleDelegator
+class ShipmentDecoratorStub < SimpleDelegator
```

Comment:
No let's keep stub, but let's change the file name to `shipment_decorator_stub.rb` since it contains only one class and is now named: `ShipmentDecoratorStub`.